README.md

```diff
@@ -25,58 +25,68 @@ _A Git template for bootstrapping a variety of projects with a consistent set of
 1. Update [CONTRIBUTING.md](CONTRIBUTING.md). For more information about open source contributions see [https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors)
 1. Delete any unwanted files in /docker (or the entire /docker folder if you're not containerising)
 1. Update [CHANGELOG.md](CHANGELOG.md)
-1. Delete this instruction block
+1. Delete this instruction block</i>
 
-# Table of Content
 
-- [Generic New Project Template](#generic-new-project-template)
-  - [Instructions](#instructions)
-- [Table of Content](#table-of-content)
 - [Prerequisites](#prerequisites)
-- [Setup and Configuration](#setup-and-configuration)
+- [Installation](#installation)
+  - [Environment File (.env) Details](#environment-file-env-details)
 - [Usage](#usage)
-- [For Users](#for-users)
-- [Change Log](#change-log)
-- [Meta](#meta)
+  - [For Local Development](#for-local-development)
+  - [To Deploy to Stage or Prod](#to-deploy-to-stage-or-prod)
 - [Contributing](#contributing)
+- [Change Log](#change-log)
 
-# Prerequisites
+## Prerequisites
 
 The versions listed for these prerequisites are current at the time of writing. More recent versions will likely work but "your mileage may vary".
 
 - A good code editor.
-- [Node.js v12.13.0 and NPM 6.14.5](https://nodejs.org/en/download/)
-- [Git 2.25](https://git-scm.com/downloads)
+- [Node.js v12.13.0 and NPM 7.0.0](https://nodejs.org/en/download/)
+- [Git 2.28](https://git-scm.com/downloads)
+
+## Installation
+
+1. Clone to repository locally, following the directory structure convention:
+
+   ```sh
+   git clone git@github.com:tforster/git-new.git \
+       ~/dev/{agent}/{client}/{product}/{project}/{component}
+   ```
 
-# Setup and Configuration
+1. Change the current working directory to the new repository directory:
 
-Clone this repository as your new project.
+   ```sh
+   cd ~/dev/{agent}/{client}/{product}/{project}/{component}
+   ```
 
-```sh
-git clone git@github.com:tforster/git-new.git /my/path/to/my/project-name
-```
+1. Install dependencies:
 
-# Usage
+   ```sh
+   npm install
+   ```
 
-- Describe how your repository should be used.
-- Include any references to other repositories such as NPM, DockerHub, etc.
+### Environment File (.env) Details
```

Comment:
Perhaps this can become **Configuration** and for specific projects it can cover other things like creating the certificates, etc.?

Reply:
```suggestion
### Configuration
```